src/styles.css

```diff
@@ -2,16 +2,32 @@
   display: none;
 }
 @media print {
+  html, body {
+    width:210mm;
+  }
```

Comment:
what is it?

Reply:
210 mm - A4 width
We explicitly tell it to prevent cases when content is overwhelmed

Reply:
It should be optional 
PrintProvider width = ‘A4’ | Number | other css values